settings.gradle

```diff
@@ -0,0 +1,168 @@
+project(':beam-sdks-parent:beam-sdks-java-parent:beam-sdks-java-extensions-parent:beam-sdks-java-extensions-json-jackson').projectDir = "$rootDir/sdks/java/extensions/jackson" as File
+project(':beam-sdks-parent:beam-sdks-java-parent:beam-sdks-java-extensions-parent:beam-sdks-java-extensions-join-library').projectDir = "$rootDir/sdks/java/extensions/join-library" as File
+project(':beam-sdks-parent:beam-sdks-java-parent:beam-sdks-java-extensions-parent:beam-sdks-java-extensions-protobuf').projectDir = "$rootDir/sdks/java/extensions/protobuf" as File
+project(':beam-sdks-parent:beam-sdks-java-parent:beam-sdks-java-extensions-parent:beam-sdks-java-extensions-sorter').projectDir = "$rootDir/sdks/java/extensions/sorter" as File
+project(':beam-sdks-parent:beam-sdks-java-parent:beam-sdks-java-extensions-parent:beam-sdks-java-extensions-sql').projectDir = "$rootDir/sdks/java/extensions/sql" as File
+project(':beam-sdks-parent:beam-sdks-java-parent:beam-sdks-java-extensions-parent').projectDir = "$rootDir/sdks/java/extensions" as File
+project(':beam-sdks-parent:beam-sdks-java-parent:beam-sdks-java-nexmark').projectDir = "$rootDir/sdks/java/nexmark" as File
+project(':beam-sdks-parent:beam-sdks-java-parent:beam-sdks-java-harness').projectDir = "$rootDir/sdks/java/harness" as File
+project(':beam-sdks-parent:beam-sdks-java-parent:beam-sdks-java-container').projectDir = "$rootDir/sdks/java/container" as File
+project(':beam-sdks-parent:beam-sdks-java-parent:beam-sdks-java-java8tests').projectDir = "$rootDir/sdks/java/java8tests" as File
+project(':beam-sdks-parent:beam-sdks-java-parent').projectDir = "$rootDir/sdks/java" as File
+project(':beam-sdks-parent:beam-sdks-python:beam-sdks-python-container').projectDir = "$rootDir/sdks/python/container" as File
+project(':beam-sdks-parent:beam-sdks-python').projectDir = "$rootDir/sdks/python" as File
+project(':beam-sdks-parent').projectDir = "$rootDir/sdks" as File
+project(':beam-runners-parent:beam-java-fn-execution').projectDir = "$rootDir/runners/java-fn-execution" as File
+project(':beam-runners-parent:beam-runners-core-construction-java').projectDir = "$rootDir/runners/core-construction-java" as File
+project(':beam-runners-parent:beam-runners-core-java').projectDir = "$rootDir/runners/core-java" as File
+project(':beam-runners-parent:beam-local-artifact-service-java').projectDir = "$rootDir/runners/local-artifact-service-java" as File
+project(':beam-runners-parent:beam-runners-reference-parent:beam-runners-reference-job-orchestrator').projectDir = "$rootDir/runners/reference/job-server" as File
+project(':beam-runners-parent:beam-runners-reference-parent').projectDir = "$rootDir/runners/reference" as File
+project(':beam-runners-parent:beam-runners-direct-java').projectDir = "$rootDir/runners/direct-java" as File
+project(':beam-runners-parent:beam-runners-flink_2.10').projectDir = "$rootDir/runners/flink" as File
```

Comment:
<!--new_thread; commit:4adedcc8ea759a78e227638bdfda2c4fc0dadbe3; resolved:0-->
The fact that we have to set this looks pretty suspicious. Probably it means we are just reaching into other project directories when we should depend on declared outputs?